distributed/protocol/numpy.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import print_function, division, absolute_import
+
+import sys
+
+import numpy as np
+
+try:
+    import blosc
+    n = blosc.set_nthreads(2)
```

Comment:
Can I interest you in raising an issue or perhaps a short PR?  Relevant files is distributed/config.py and an example is here:

    distributed/scheduler.py:BANDWIDTH = config.get('bandwidth', 100e6)